build/Dockerfile

```
@@ -121,58 +121,85 @@ CMD /bin/bash
 # STAGE: devnet-base
 #
 # This stage installs Foundry.
-FROM ${BASE_IMAGE} as devnet-base
+FROM golang:${GO_VERSION}-bookworm as devnet-base
 
 # Install system dependencies.
-ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
 set -e
 apt-get update
-apt-get install -y --no-install-recommends \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

If use ARG, remove this here.

```suggestion
apt-get install -y --no-install-recommends \
```